packages/package.json

```diff
@@ -39,13 +39,14 @@
     "object-assign": "^4.1.1",
     "prop-types": "^15.7.2",
     "react-autosuggest": "^9.4.3",
-    "react-responsive": "^6.1.2",
+    "react-responsive": "^8.2.0",
     "react-slider": "^1.3.1",
     "react-transition-group": "^2.5.3",
-    "react-virtualized": "^9.22.3",
-    "react-window": "^1.8.7"
+    "react-window": "^1.8.7",
+    "react-table": "^7.8.0"
```

Comment:
minor nitpick: Can you move this up above `react-transition-group` to keep it alphabetical?